app/controllers/sac_cas/json_api/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module SacCas::JsonApi::EventsController
+  extend ActiveSupport::Concern
+
+  WEBSITE_TOKEN = "[Hitobito-Typo3]"
+  EXPIRES_IN = 3.hours
+
+  def index
+    return super unless from_website?
+
+    authorize!(:list_available, Event)
+    data = Rails.cache.fetch(dynamic_cache_key, expires_in: EXPIRES_IN) do
+      resource_class.all(params).to_jsonapi
```

Comment:
Ist das identisch mit dem Code in `super` (ich denke an die Abilities)? Allenfalls könnte der token/user Name noch in den cache_key aufgenommen werden, damit das Caching gleich für alle Tokens/User funktioniert. Sonst gibt es garantiert ein Problem, wenn sie mal ein neues Token ausstellen (was ja security-mässig hie und da sinnvoll wäre).